### DISCUSSION TITLE
How to correctly delete files in onedrive

### DISCUSSION THREAD

**Discussion by KGarvey92** [remotely-save/remotely-save#360] [Q&A]
I have my vault synced with remotely-save and for the most part it works great. Occasionally there will be a small sync issue, but it will just create an extra file and I can sort it out no problem. So thanks for the app, it's a lifesaver for me syncing across devices in different 'ecosystems'. 

I am however having an issue when deleting notes from my mac. When I delete a note and the 'send to system trash' message comes up, rather than actually doing that, it just moves the file to the root of my Onedrive directory. Is this happening for anyone else, or does anyone have any ideas how to get around it? It doesn't appear to happen on my iphone or windows.

**Comment by fyears** [comment]
a new version two weeks later should fix this problem
